src/test/java/bio/terra/common/auth/AuthService.java

```diff
@@ -24,7 +24,7 @@
   // this should always match exactly what the UI requests, so our tests represent actual user
   // behavior:
   private List<String> userLoginScopes =
-      Arrays.asList("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-platform");
+      Arrays.asList("openid", "email", "profile", "https://www.googleapis.com/auth/cloud-billing");
```

Comment:
If this has the same meaning as `AuthenticationUtils.userLoginScopes` it would be better to make that constant public and use it here instead of duplicating it.